projects/gateway2/helm/gloo-gateway/templates/gateway/proxy-deployment.yaml

```diff
@@ -439,45 +439,8 @@ data:
       metadata:
         role: gloo-kube-gateway-api~{{ $gateway.gatewayNamespace }}~{{ $gateway.gatewayNamespace }}-{{ $gateway.gatewayName | default (include "gloo-gateway.gateway.fullname" .) }}
     static_resources:
-      listeners:
-      - name: readiness_listener
-        address:
-          socket_address: { address: 0.0.0.0, port_value: 8082 }
-        filter_chains:
-          - filters:
-            - name: envoy.filters.network.http_connection_manager
-              typed_config:
-                "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
-                stat_prefix: ingress_http
-                codec_type: AUTO
-                route_config:
-                  name: main_route
-                  virtual_hosts:
-                    - name: local_service
-                      domains: ["*"]
-                      routes:
-                        - match:
-                            path: "/ready"
-                            headers:
-                              - name: ":method"
-                                string_match:
-                                  exact: GET
-                          route:
-                            cluster: admin_port_cluster
-                http_filters:
-{{- if $gateway.readinessProbe }}
-                  - name: envoy.filters.http.health_check
-                    typed_config:
-                      "@type": type.googleapis.com/envoy.extensions.filters.http.health_check.v3.HealthCheck
-                      pass_through_mode: false
-                      headers:
-                      - name: ":path"
-                        exact_match: "/envoy-hc"
```

Comment:
this scares me i suspect users have configured instances where this is used for start up even pre readiness and not having this until translation may cause over eager setups to die

Reply:
In that case, it isn't a true readiness check, right? The liveness probe can still be used or a custom readiness probe can be used to point to the admin endpoint